### PR TITLE
gcp: add focal PRO test for enabling fips

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
gcp: add focal GCP PRO test for enabling fips

PS: we need to update that test after we have updated the contract definition for FIPS on focal GCP PRO

## Test Steps
Run the new integration test

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [ ] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: features/ubuntu_pro.feature
Comment (CalvoM) [inline features/ubuntu_pro.feature:709]: Just a minor formatting issue, an accidental identation.

**Comment by CalvoM** [comment]
Hey @lucasmoura considering there is an `ubuntu_pro_fips.feature` file, couldn't this be added to that file, or maybe merge the two files?

**Comment by renanrodrigo** [comment]
Since we want this to land soon in the contract side (it is about to, isn't it?) and it is just the test, we may want to have it ready without the overrides and wait for the change to land there before merging. What do you think?

File: features/ubuntu_pro.feature
Comment (lucasmoura) [reply features/ubuntu_pro.feature:709]: Sorry @CalvoM I think I am missing your point here

Comment (orndorffgrant) [reply features/ubuntu_pro.feature:709]: @lucasmoura The 30 lines above this are all indented too much :)

Comment (lucasmoura) [reply features/ubuntu_pro.feature:709]: Oh, now I see it :facepalm: Thanks for spotting that @CalvoM and @orndorffgrant

**Comment by lucasmoura** [comment]
@renanrodrigo @orndorffgrant @CalvoM Tests updated

File: features/enable_fips_vm.feature
Comment (renanrodrigo) [inline features/enable_fips_vm.feature:137]: WIP slipped away

File: features/enable_fips_vm.feature
Comment (renanrodrigo) [inline features/enable_fips_vm.feature:251]: Why should this part be commented out?

File: features/ubuntu_pro.feature
Comment (renanrodrigo) [inline features/ubuntu_pro.feature:617]: Now that this is moved to the other file and without the overrides, this part can be removed, right?

**Review by renanrodrigo — CHANGES_REQUESTED** [review]
Tests passing for me, but I think there are some changes needed in the structure

**Comment by lucasmoura** [comment]
@renanrodrigo updated

**Review by renanrodrigo — APPROVED** [review]
LGTM

**Review by orndorffgrant — APPROVED** [review]
This worked for me!

One question: Should these tests also run for Bionic? @lucasmoura

**Comment by lucasmoura** [comment]
@orndorffgrant Good catch. I will add the bionic coverage

**Review by orndorffgrant — APPROVED** [review]
@lucasmoura Thanks! Bionic tests also passed for me. I think this just needs a rebase and is good to go!

**Review by renanrodrigo — APPROVED** [review]

**Pull request merged**
